WordPress/src/main/java/org/wordpress/android/ui/prefs/SiteSettingsFragment.java

```diff
@@ -384,9 +382,8 @@ public boolean onPreferenceClick(Preference preference) {
         } else if (preference == mCategoryPref || preference == mFormatPref) {
             return !shouldShowListPreference((DetailListPreference) preference);
         } else if (preference == mExportSitePref) {
-            showExportContentDialog();
```

Comment:
Ah, that's my fault. I didn't want to add too many features in a single PR. I removed all export content work and will be doing it as a seperate PR. Good eye! :eyes: